docs/source/conf.py

```diff
@@ -50,7 +50,7 @@
 ]
 
 # MyST
-myst_enable_extensions = ["colon_fence", "deflist", "dollarmath"]
+myst_enable_extensions = ["dollarmath"]
```

Comment:
```suggestion
myst_enable_extensions = ["amsmath"]
```

While browsing things earlier today, I noted that dollarmath might be gone from MyST in the future https://github.com/executablebooks/MyST-Parser/pull/505. Makes sense because for parsers, you definitely want to differentiate between start/end sequences delimiting environments. So we'll want to be future-proof here :-)

Reference on AMSmath: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-amsmath

Reply:
We would probably have to remove it completely then and only use the math-directive for equations. Because dollarmath is the only extension that works with _mdformat_ (for legacy reasons, might also be changed in the future).

Reply:
Yes, let's remove it then. Thanks!